src/alexjs/alex.handler.ts

```diff
@@ -67,7 +67,7 @@ export class AlexHandler {
         targetNotification.notificationId,
       );
       if (notificationMessage && notificationMessage.deletable) {
-        await notificationMessage.delete();
+        await notificationMessage.delete().catch(console.error);
```

Comment:
what about wrapping these in a `try/catch`?

Reply:
but idk, why will you wrap a promise in try/catch if you have access to the catch function?

Reply:
here the styles are mixed. The `.catch` is used when using `then` but we are using `await` so the `try/catch` should be used

Reply:
Fixed in https://github.com/EddieHubCommunity/EddieBot/pull/605/commits/65dcab71b171fcd4710c7569f81d4c5f5c5a7a80